static/app/components/actions/resolve.tsx

```diff
@@ -244,10 +244,8 @@ function ResolveActions({
         <ResolveButton
           priority={priority}
           size={size}
-          title={t(
-            'Resolves the issue. The issue will get unresolved if it happens again.'
-          )}
-          tooltipProps={{delay: 300, disabled: disabled || disableTooltip}}
+          title={t("We'll nag you with a notification if the issue's seen again.")}
```

Comment:
```suggestion
          title={t("Marks the issue resolved. We'll nag you with a notification if it's seen again.")}
```

IMHO, it'd be nice to have parallel sentence structure between the two tooltips.

Reply:
We shouldn’t have a button that says `Resolved` and the label description then duplicates the button text. The most important thing they need to know here is that they’ll get a notification if it regresses. Let's keep it as is for now.